configuration/iam-service/variables.env

```diff
@@ -6,3 +6,5 @@ ADMINISTRATOR_USERNAME=qpsdemo
 ADMINISTRATOR_PASSWORD=qpsdemo
 ADMINISTRATOR_GROUP=Super admins
 OWNER_GROUP=Admins
+
+REPORTING_SERVICE_URL=http://localhost
```

Comment:
1. I believe this is not the reporting service url. This looks more like Zebrunner's host.
2. As far as I remember, hulu team deploys Zebrunner in their private cloud. Therefore, the host value for them is not `localhost`. Probably it makes sense to specify such a property in other more common place and not hide it in service specific properties.